TempoCore/Source/TempoTime/Private/TempoTimeServiceSubsystem.cpp

```diff
@@ -61,6 +63,11 @@ void UTempoTimeServiceSubsystem::SetTimeMode(const TempoTime::TimeModeRequest& R
 		}
 	}
 
+	if (!Cast<ATempoTimeWorldSettings>(GetWorld()->GetWorldSettings()))
+	{
+		ResponseContinuation.ExecuteIfBound(TempoEmpty(), grpc::Status(grpc::StatusCode::FAILED_PRECONDITION, "SetTimeMode has no effect unless using TempoTimeWorldSettings"));
```

Comment:
missing `return`

Reply:
Also, do this before setting the setting. The user would assume the setting has not been set when receiving and error.